test/e2e/specs/__snapshots__/block-transforms.test.js.snap

```diff
@@ -0,0 +1,297 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`test transforms individual transforms work as expected Should correctly transform block in fixture core__audio to File block 1`] = `
```

Comment:
Is there any way to make this more descriptive and concise? Something like: "Paragraph block should transform to Heading block"?

Reply:
We can do that, but the thing is that we have many tests testing Paragraph to Heading block (one for each paragraph fixture), that was the reason I added the fixture to make it easier to debug in case the test fails. Maybe adding also the block name is a good trade-off.

Reply:
Ok, we can always iterate on this.